modules/dr-restoring-cluster-state.adoc

```diff
@@ -186,6 +186,186 @@ etcd-ip-10-0-143-125.ec2.internal                1/1     Running     1
 +
 If the status is `Pending`, or the output lists more than one running etcd pod, wait a few minutes and check again.
 
```

Comment:
Just checking that you got confirmation that this is the correct place in the procedure to add this new step